app/src/components/tables/BlockTable/BlocksTable.tsx

```diff
@@ -216,10 +234,32 @@ export const BlocksTable: React.FC<BlocksTableProps> = ({
       tableBodyLoading={isTableLoading}
       currentPageSize={blocksTableOptions.pagination.pageSize}
       placeholderData={{
-        header: { height: 0 },
+        header: { height: 0, era_id: 0, timestamp: '2023-06-05T17:06:44.864Z' },
+        body: {
+          proposer:
+            '017d96b9a63abcb61c870a4f55187a0a7ac24096bdb5fc585c12a686a4d892009e',
+          deploy_hashes: [],
+          transfer_hashes: [],
+        },
+        hash: '52f7c16323868f73343335f26a484aed0067a3e769dc9187dbae6a305e2b59f3',
```

Comment:
Again, still getting back up to speed with this codebase.  What is this placeholder data for?  

Thoughts on slapping this data in a separate file?

Reply:
There's a comment left on the prop in the `Table.tsx` component if you want to update yourself on that. We can also hop on a call if that doesn't make it clear.

Reply:
I'm fine either way. I'll leave it up to you if you want me to move to a separate file.

Reply:
Thanks -- I'll take a look at the comment :)

And let's just leave the code as is for now